cypress/integration/reports.spec.ts

```diff
@@ -16,7 +16,7 @@ context('Reports Integration Test', () => {
     indexedDB.deleteDatabase('OpenCRVS')
   })
 
-  it('Tests for performance reports birth using minimum input', () => {
+  it.skip('Tests for performance reports birth using minimum input', () => {
```

Comment:
I skipped these tests as I think they might not be valid anymore. @azantor64 can you confirm?